lighthouse-viewer/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
should we extend our base or not worth it cause we're just checking the src here?

Reply:
> should we extend our base or not worth it cause we're just checking the src here?

it doesn't buy us a whole lot because we have to overwrite `typeRoots` and `include`, but it's kind of nice to share the other options. Done.